### DISCUSSION TITLE
please make it compactible to all version

### DISCUSSION THREAD

**Discussion by Samith71** [GriefPrevention/GriefPrevention#1289] [General]
Hi I am running a mc server and i want grief prevention on my server but my server is on 1.13.2. so i am having problem please make it compactible with all versions of minecraft

**Comment by bigpresh** [comment]
The most recent Grief Prevention version that will work with ancient 1.13.x servers is [16.11.6](https://github.com/TechFortress/GriefPrevention/releases/tag/16.11.6), that's the version you'll need to use.
